web/src/pages/Dashboard/styles.css

```diff
@@ -0,0 +1,122 @@
+#page-landing {
```

Comment:
Would be nice to:
fix logo,
change colors (too dark)
style buttons with animations